services/api-server/src/simcore_service_api_server/models/schemas/meta.py

```diff
@@ -9,16 +9,16 @@ class Meta(BaseModel):
     released: dict[str, VersionStr] | None = Field(
         None, description="Maps every route's path tag with a released version"
     )
-    docs_url: AnyHttpUrl = Field(default="https://docs.osparc.io")
-    docs_dev_url: AnyHttpUrl = Field(default="https://api.osparc.io/dev/docs")
+    docs_url: AnyHttpUrl
+    docs_dev_url: AnyHttpUrl
```

Comment:
are you sure about this change? You removed the defaults.

Reply:
yes :-)